Updated tex

shujaatkhan opened the pull request:
I have updated the tex file and updated the PDFs as well.
Could you please run the scrip you have, so that an updated html page can be created?

llorracc-git commented:
OK, I've run my script.  Could you check to make sure everything worked as
it should have?

PS.  Thanks for your work over the holiday!



On Wed, Nov 23, 2016 at 12:44 PM, shujaatkhan <notifications@github.com>
wrote:

> I have updated the tex file and updated the PDFs as well.
> Could you please run the scrip you have, so that an updated html page can
> be created?
> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/llorracc-git/MainPage/pull/5
> Commit Summary
>
>    - Merge remote-tracking branch 'llorracc-git/master'
>    - Merge remote-tracking branch 'llorracc-git/master'
>    - Adding interfolio
>    - Merge remote-tracking branch 'llorracc-git/master'
>    - Merge remote-tracking branch 'llorracc-git/master'
>    - Merge remote-tracking branch 'origin/master'
>    - Merge remote-tracking branch 'llorracc-git/master'
>    - Updated RecLetters.tex
>    - Updated PDF in the main directory
>
> File Changes
>
>    - *M* RecLetters.pdf
>    <https://github.com/llorracc-git/MainPage/pull/5/files#diff-0> (0)
>    - *A* RecLetters/RecLetters.pdf
>    <https://github.com/llorracc-git/MainPage/pull/5/files#diff-1> (0)
>    - *M* RecLetters/RecLetters.tex
>    <https://github.com/llorracc-git/MainPage/pull/5/files#diff-2> (226)
>
> Patch Links:
>
>    - https://github.com/llorracc-git/MainPage/pull/5.patch
>    - https://github.com/llorracc-git/MainPage/pull/5.diff
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/llorracc-git/MainPage/pull/5>, or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AGRCJCRVdO_iVyOYMqOTLUqw0QOqvnJhks5rBHtigaJpZM4K630o>
> .
>



-- 
- Chris Carroll

shujaatkhan commented:
Perfect. Now it’s fixed.

From: llorracc-git <notifications@github.com>
Reply-To: llorracc-git/MainPage <reply@reply.github.com>
Date: Wednesday, November 23, 2016 at 9:51 PM
To: llorracc-git/MainPage <MainPage@noreply.github.com>
Cc: Shujaat Khan <sakhan@jhu.edu>, Author <author@noreply.github.com>
Subject: Re: [llorracc-git/MainPage] Updated tex (#5)

OK, I've run my script. Could you check to make sure everything worked as
it should have?

PS. Thanks for your work over the holiday!



On Wed, Nov 23, 2016 at 12:44 PM, shujaatkhan <notifications@github.com>
wrote:

> I have updated the tex file and updated the PDFs as well.
> Could you please run the scrip you have, so that an updated html page can
> be created?
> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
> https://github.com/llorracc-git/MainPage/pull/5
> Commit Summary
>
> - Merge remote-tracking branch 'llorracc-git/master'
> - Merge remote-tracking branch 'llorracc-git/master'
> - Adding interfolio
> - Merge remote-tracking branch 'llorracc-git/master'
> - Merge remote-tracking branch 'llorracc-git/master'
> - Merge remote-tracking branch 'origin/master'
> - Merge remote-tracking branch 'llorracc-git/master'
> - Updated RecLetters.tex
> - Updated PDF in the main directory
>
> File Changes
>
> - *M* RecLetters.pdf
> <https://github.com/llorracc-git/MainPage/pull/5/files#diff-0> (0)
> - *A* RecLetters/RecLetters.pdf
> <https://github.com/llorracc-git/MainPage/pull/5/files#diff-1> (0)
> - *M* RecLetters/RecLetters.tex
> <https://github.com/llorracc-git/MainPage/pull/5/files#diff-2> (226)
>
> Patch Links:
>
> - https://github.com/llorracc-git/MainPage/pull/5.patch
> - https://github.com/llorracc-git/MainPage/pull/5.diff
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/llorracc-git/MainPage/pull/5>, or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AGRCJCRVdO_iVyOYMqOTLUqw0QOqvnJhks5rBHtigaJpZM4K630o>
> .
>



--
- Chris Carroll


—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/llorracc-git/MainPage/pull/5#issuecomment-262678615>, or mute the thread<https://github.com/notifications/unsubscribe-auth/ATHpCR4kdMuImYkv7-yjdsKXDdt6Wlpeks5rBPvOgaJpZM4K630o>.
